modules/gitrepo/branch.go

```diff
@@ -0,0 +1,33 @@
+// Copyright 2024 The Gitea Authors. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+package gitrepo
+
+import (
+	"context"
+
+	repo_model "code.gitea.io/gitea/models/repo"
```

Comment:
Ah yep, this file should be a part of `models` or even `services`

Reply:
Changed in 18b26289f2881ec9f23b0f249da5f82a1b6e5cba